radixdlt/babylon-wallet-android
# ABW-2931 resolve assets in stake/pool transactions

## Description
- now we expect other assets present in staking/unstaking/claiming,pool contributions/redemptions transactions
- used use case provided by Michale for assets resolution and simplified logic of processors
- I've moved some of the aggregation logic, so now ui is driven by manifest summary
- aggregation is turned off for now, to not introduce even more change and potential issues

## How to test

- run through sandbox integration tests
- run through all staking/pool tx types via sandbox
- use staking/pool raw manifests to append other asset transfers to those transaction types. You can copy manifest from the app, append withdraw/deposit instructions and verify manifest preview correctly renders those. For example you can append such lines for token transfer along the stake manifest:
`CALL_METHOD
    Address("your_account_1")
    "withdraw"
    Address("resource_tdx_2_1tknxxxxxxxxxradxrdxxxxxxxxx009923554798xxxxxxxxxtfd2jc") <-- resource address
    Decimal("2")
;
TAKE_FROM_WORKTOP
    Address("resource_tdx_2_1tknxxxxxxxxxradxrdxxxxxxxxx009923554798xxxxxxxxxtfd2jc") <-- resource address
    Decimal("2")
    Bucket("bucketX")
;
CALL_METHOD
    Address("your_account_2")
    "deposit"
    Bucket("bucketX")
;`

## PR submission checklist
- [x] I have run through sandbox integration tests
- [x] I have tested staking/pool tx types, as well as all of those with additional resource transfers
